packages/epo-react-lib/src/assets/locales/es/translation.json

```diff
@@ -1 +1 @@
-{}
+{ "translation": {} }
```

Comment:
stub out this translation file so it can be merged into the widget libs translations